CI appears broken on `main`

All commits merged to `main` are failing since Feb 26.
They fail with
```
21:05:37  ==> erlfdb (compile)
21:05:38  Compiling c_src/atoms.c
21:05:38  Compiling c_src/main.c
21:05:38  In file included from c_src/main.c:18:
21:05:38  c_src/fdb.h:17:10: fatal error: 'foundationdb/fdb_c.h' file not found
```
which is weird, because https://github.com/apache/couchdb-ci/blob/main/bin/apt-dependencies.sh#L190 installs FoundationDB just like https://github.com/apache/couchdb-config/blob/main/.travis.yml .
(And branch / PR CI looks fine)

--- a/build-aux/Jenkinsfile.full
+++ b/build-aux/Jenkinsfile.full
@@ -498,6 +498,7 @@ pipeline {
         } // stage
 
         stage('Debian Buster arm64v8') {
+          when { expression { return false } }
           agent {
             docker {
               image 'couchdbdev/arm64v8-debian-buster-erlang-20.3.8.25-1:latest'

--- a/build-aux/Jenkinsfile.full
+++ b/build-aux/Jenkinsfile.full
@@ -703,7 +703,8 @@ pipeline {
           sh '''
             cp js/centos-7/*rpm pkgs/centos7
             cp js/centos-8/*rpm pkgs/centos8
-            cd ../centos7 && createrepo --database .
+            cd pkgs
+            cd centos7 && createrepo --database .
             cd ../centos8 && createrepo --database .
           '''
 

--- a/test/elixir/test/changes_async_test.exs
+++ b/test/elixir/test/changes_async_test.exs
@@ -93,7 +93,7 @@ defmodule ChangesAsyncTest do
 
     create_doc_bar(db_name, "bar")
 
-    changes = process_response(req_id.id, &parse_event/1, 3000)
+    changes = process_response(req_id.id, &parse_event/1, 5000)
 
     assert length(changes) == 2
     assert Enum.at(changes, 0)["id"] == "foo"
@@ -174,7 +174,7 @@ defmodule ChangesAsyncTest do
 
     req_id =
       Rawresp.get(
-        "/#{db_name}/_changes?feed=continuous&filter=changes_filter/bop&timeout=1000",
+        "/#{db_name}/_changes?feed=continuous&filter=changes_filter/bop&timeout=2000",
         stream_to: self(),
         direct: worker_pid
       )
@@ -216,7 +216,7 @@ defmodule ChangesAsyncTest do
     :ok = wait_for_headers(req_id.id, 200)
     create_doc(db_name, %{_id: "doc3", value: 3})
 
-    changes = process_response(req_id.id, &parse_changes_line_chunk/1, 3000)
+    changes = process_response(req_id.id, &parse_changes_line_chunk/1, 5000)
 
     changes_ids =
       changes

